app/uk/gov/hmrc/exports/services/email/EmailSender.scala

```diff
@@ -37,7 +37,10 @@ class EmailSender @Inject()(
     obtainEori(mrn).flatMap {
       case Some(eori) =>
         obtainEmailAddress(eori).flatMap {
-          case Some(verifiedEmailAddress) => sendEmail(mrn, eori, verifiedEmailAddress)
+          case Some(Email(email, true)) => sendEmail(mrn, eori, Email(email, deliverable = true))
+          case Some(Email(_, false)) =>
+            logger.warn(s"Email address for EORI: [$eori] was not deliverable")
+            Future.successful(MissingData)
```

Comment:
You could simplify these lines as: 
```
case Some(email) if email.deliverable => sendEmail(mrn, eori, email)
case Some(_) => 
            logger.warn(s"Email address for EORI: [$eori] was not deliverable")
            Future.successful(MissingData)
```
This would prevent you having to deconstructing the Email case class just to re-construct it again :wink: